CreationalDesignPatterns/AbstractFactory/CoffeeFactory/Macchiato.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace CoffeeFactory
+{
+    public class Macchiato : MilkCoffee
+    {
+        public override string Name
+        {
+            get { return "Macchiato"; }

```

Comment:
We can use nameof(Macchiato)

Reply:
Constants instead? As for the others

Reply:
public const string DisplayMacchiato = "Macchiato";

Reply:
Fine!